client/components/settings/EmailsParameters.js

```diff
@@ -5,22 +5,24 @@ import Reports from './Reports';
 
 export default class EmailsParameters extends React.Component {
     render() {
-        return <div>
-            <Alerts
-                alertType="balance"
-                sendIfText={$t('client.settings.emails.send_if_balance_is')}
-                titleTranslationKey="settings.emails.add_balance"
-                panelTitleKey='settings.emails.balance_title'
-            />
+        return (
+            <div>
+                <Alerts
+                  alertType="balance"
+                  sendIfText={ $t('client.settings.emails.send_if_balance_is') }
+                  titleTranslationKey="settings.emails.add_balance"
```

Comment:
pre-existing: this should contain `client` as the prefix, it should not be included in the component below.